documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -171,29 +177,7 @@ Example updates:
 
 ### Upgrade the operator
 
-You can upgrade a 2.6 or 3.x operator while the operator's domain resources are deployed and running.
-
-#### Upgrading a 2.6 operator to a 3.x operator
-
-{{% notice note %}}
-Because operator 3.0.0 introduces _non-backward compatible_ changes, you cannot use `helm upgrade` to upgrade
-a 2.6.0 operator to a 3.x operator. Instead, you must delete the 2.6.0 operator and then install the
-3.x operator.
-{{% /notice %}}
-
-The deletion of the 2.6.0 operator will _not affect_ the Domain CustomResourceDefinition (CRD) and will _not stop_ any
-WebLogic Server instances already running.
-
-When the 3.x operator is installed, it will automatically roll any running WebLogic Server instances created by the 2.6.0 operator.
-This rolling restart will preserve WebLogic cluster availability guarantees (for clustered members only) similarly to any other rolling restart.
-
-To delete the 2.6.0 operator:
-
-```text
-$ helm delete weblogic-operator -n weblogic-operator-namespace
-```
-
-Then install the 3.x operator using the [install the operator](#install-the-operator) instructions.
+You can upgrade a 3.x operator while the operator's domain resources are deployed and running.
 
 #### Upgrading a 3.x operator
```

Comment:
In my opinion, this also needs a discussion of choosing and setting the helm chart version because we are choosing to "hide" the existence of the operator image as a simplification, which is what is being used to set the version today.

It might also be nice to teach folks how to quickly determine the current running version since we are "hiding" discussion of the operator image, which is the current way folks figure that out.

Reply:
@rjeberhard The new simplification ties helm chart version to operator version, right? I assume this is OK?  Or are there cases when customers want to use the latest helm chart but an older version of the operator image?

Reply:
I can make specific edits, but if sections need more info or elaboration, please add the description to the JIRA [OWLS-97331](https://jira.oraclecorp.com/jira/browse/OWLS-97331).

Reply:
Oh, I think I understand... the upgrade use case is a special case because we've said "--reuse-values" which would preserve the image name from the original install. The customer could alternatively not use "--reuse-values" but instead provide the arguments from the original install or that the customer now wants.

Reply:
I am confused.  We can combine `—reuse-values` with a set of the image to point to the new version.  Is there some reason that this isn’t desirable or won’t work?  We definitely want to make things easy but for the upgrade use case, I feel like the combination of `—reuse-values` and `—set` to specify the new image name is the most understandable mechanism.

Reply:
Retracting my comment based on our discussion in Slack.